client/client-multi/client-objective-c/KaaTests/logging/DefaultLogUploadStrategyTest.m

```diff
@@ -153,7 +155,7 @@ - (void)maxParallelUploadCountInSyncRequestHelper:(int64_t)maxParallelUploads {
     [given([executorContext getApiExecutor]) willReturn:apiExecutor];
     [given([executorContext getSheduledExecutor]) willReturn:schedulerQueue];
     
-    AbstractLogCollector *logCollector = [[DefaultLogCollector alloc] initWithTransport:logTransport executorContext:executorContext channelManager:channelManager failoverManager:failoverManager];
+    AbstractLogCollector *logCollector = [[NoTimeoutLogCollector alloc] initWithTransport:logTransport executorContext:executorContext channelManager:channelManager failoverManager:failoverManager];
```

Comment:
Is there any reason to use AbstractLogCollector pointer instead of NoTimeoutLogCollector?